docs/http-api/persistent.md

```diff
@@ -118,64 +124,62 @@ For example:
-|---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----|
-| `stream`            | The stream to the persistent subscription is on.                                                                                                                                                                                     |     |
-| `subscription_name` | The name of the subscription group.                                                                                                                                                                                                  |     |
-| `action`            | <ul><li>**Park**: Don't retry the message, park it until a request is sent to reply the parked messages</li><li>**Retry**: Retry the message</li><li>**Skip**: Discard the message</li><li>**Stop**: Stop the subscription</li></ul> |     |
-| `messageid`         | The id of the message that needs to be acked                                                                                                                                                                                         |     |
+| Parameter           | Description                                                                                                                                                                                                                               |
+| ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `stream`            | The stream to the persistent subscription is on.                                                                                                                                                                                          |
+| `subscription_name` | The name of the subscription group.                                                                                                                                                                                                       |
+| `action`            | <ul><li>**Park**: Don't retry the message; park it until a request is sent to replay the parked messages.</li><li>**Retry**: Retry the message.</li><li>**Skip**: Discard the message.</li><li>**Stop**: Stop the subscription.</li></ul> |
+| `messageid`         | The ID of the message that needs to be Nacked.                                                                                                                                                                                            |
```

Comment:
```suggestion
| `messageid`         | The ID of the message that needs to be acked.                                                                                                                                                                                            |
```

Reply:
@cdevarenne do we need to define it as "needs to be acked" or can we use "needs to be nacked" since that's what we are doing in this query?